components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -91,6 +91,12 @@ impl<'a, 'b> Cli<'a, 'b> {
     pub fn add_runtime_args(self) -> Self {
         Cli {
             app: self.app
+                .arg(
+                    Arg::with_name("K8S_NAME")
+                        .value_name("K8S_NAME")
+                        .long("k8s-name")
+                        .help("The Kubernetes resource name"),
```

Comment:
Maybe like you should mention the default is pkg_name + version? WDYT?

Reply:
Sure